src/app/page.tsx

```diff
@@ -1,14 +1,106 @@
-import React from "react";
+"use client"
 
+import React, { useEffect, useState } from "react";
+import { SupplierCard } from '@/components/mols';
+import { CategoryCard } from '@/components/atoms'
+import SupLogo from '@/images/easytolive/logo/logotipo-fundoazulroxo.svg'
```

Comment:
SupplierLogo. 

We make code for others. If u use generic therms the code can be so hard for reading by other people

Reply:
really nice point... i have to think more like that